ui/src/tasks/actions/v2/index.ts

```diff
@@ -390,7 +396,7 @@ export const importScript = (script: string, fileName: string) => async (
     dispatch(notify(taskImportSuccess(fileName)))
   } catch (error) {
     console.error(error)
-    const message = getDeep<string>(error, 'message', '')
+    const message = _.get(error, 'response.data.error.message', '')
```

Comment:
same here

Reply:
Same as above. It calls the same post api for tasks